packages/interactor/test/create-interactor.test.ts

```diff
@@ -24,7 +24,7 @@ const Div = createInteractor('div')
 
 const Details = createInteractor<HTMLDetailsElement>('details')
   .selector('details')
-  .locator((element) => element.querySelector('summary')?.innerText || '')
+  .locator((element) => element.querySelector('summary')?.textContent || '')
```

Comment:
In cases like these, I've just opted to use `textContent` directly since that way we don't have to couple anything more to the `innerText` helper, and because these interactors only exist inside our test suite which is known to be JSDOM